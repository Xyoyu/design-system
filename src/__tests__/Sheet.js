import React from 'react'
import renderer from 'react-test-renderer'
import { Sheet } from '..'

describe('Sheet', () => {
  test('renders', () => {
    const json = renderer.create(<Sheet />).toJSON()
    expect(json).toMatchSnapshot()
    expect(json).toHaveStyleRule('background-color', 'rgba(255,255,255,0.875)')
  })
})
