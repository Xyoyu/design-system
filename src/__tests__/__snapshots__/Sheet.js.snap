// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sheet renders 1`] = `
.c3 {
  margin-left: auto;
  margin-right: auto;
  padding: 18px;
  width: 100%;
  background-color: rgba(255,255,255,0.875);
}

.c2 {
  max-width: 72rem;
}

.c1 {
  border-radius: 4px;
}

.c0 {
  position: relative;
  overflow: hidden;
  border-radius: 8px;
  box-shadow: 0 8px 32px rgba(0,0,0,0.0625);
}

@media screen and (min-width:32em) {
  .c3 {
    padding: 24px;
  }
}

<div
  className="c0 c1 c2 c3"
  width={1}
/>
`;
